shell.php

```diff
@@ -241,6 +257,52 @@ function _onShellCmdKeyDown(event) {
                             eShellCmdInput.value = commandHistory[historyPosition];
                         }
                         break;
+                    case 'Tab':
+                        event.preventDefault();
+                        var currentCmd = eShellCmdInput.value.split(' ');
+                        if (currentCmd.length > 0) {
```

Comment:
This will always be true, because when you split an empty string, it gives you `[""]`